File: Problems/035_Search_Insert_Position/Search_Insert_Position.cs
using System;

public class Solution {
    public int SearchInsert(int[] nums, int target)
	{
		int i;
		
		for ( i = 0; i < nums.Length; i++ ) {
			if ( nums[i] < target ) {
				continue;
			}
			else if ( nums[i] >= target ) {
				return i;
			}
		}
		
		return i;
	}

	public void Main()
	{
			Console.Write( SearchInsert(new int[] { 1, 3, 5, 6 }, 5 ) );
	}
}

File: Problems/0035_Search_Insert_Position/Search_Insert_Position.c
#include <stdio.h>

int searchInsert(int* nums, int numsSize, int target)
{
	int i;
	
	for ( i = 0; i < numsSize; i++ ) {
		if ( nums[i] < target ) {
			continue;
		}
		else if ( nums[i] >= target ) {
			return i;
		}
	}
	
	return i;
}


void main(void)
{
	int data[] = { 1, 3, 5, 6 };
	printf("%d\n", searchInsert(data, 4, 5) );
}
